src/components/dashboard/ProductsTable.tsx

```
@@ -0,0 +1,419 @@
+}: ProductsTableProps) {
+  const [searchTerm, setSearchTerm] = useState('')
+  const [sortField, setSortField] = useState<SortField>('receita_brl')
+  const [sortDirection, setSortDirection] = useState<SortDirection>('desc')
+
+  // Filter and sort data
+  const processedData = useMemo(() => {
+    let filtered = data
+
+    // Apply search filter
+    if (searchTerm) {
+      filtered = data.filter(item =>
```

ESLint

[error] Insert `⏎········`

(prettier/prettier)